services/docker-compose.devel.yml

```diff
@@ -11,6 +11,7 @@ services:
       - SC_BOOT_MODE=debug
       - LOG_LEVEL=DEBUG
       - DEBUG=true
+      - API_SERVER_PROFILING=${API_SERVER_PROFILING:-0}
```

Comment:
Please avoid defaults as a rule of thumb, at the size of oSparc it is easy to get lost in defaults and the many places they might be defined in. That can turn debugging into a nightmare. It is better to require/provide a long but comprehensive config(file) and fail gracefully otherwise, if settings aren't defined.  You can use the `.env-devel` and `repo.config` for this.

These are at least my 2cents...

Reply:
So you mean that you prefer I set a value for these env vars in all `repo.config` files? My reason for doing it like this is that this env var should take the value `0` on all deployments except the master deployments and the local deployment. So I found this solution simpler. But if you prefer it otherwise then I can change it. But then I guess you would also prefer I don't give this variable a default value in the python code so that pydantic will throw a validation error if the env var is not set, correct?

Reply:
OK, I will make the settings/envvars compulsary and modify in all the repo.configs